Changelog.md

```diff
@@ -1,3 +1,8 @@
+Unreleased
+---------------------------
+
+* [Enhancement] Enable overriding 'delete_age' via XBlock attribute
```

Comment:
You'll need to elaborate more here. In particular, you'll want to expand on the days (in configuration settings) vs. seconds (in XBlock settings) discrepancy, why it exists, and how it will go away in the future.